src/models/HourlyForecast.tsx

```diff
@@ -0,0 +1,21 @@
+import Weather from './Weather';
+import City from './City';
+
+export default class HourlyForecast {
+
+    city: City;
+    coord: { lat: number, lon: number};
+    code: number;
+    message: number;
+    cnt: number;
+    list: Weather[];
+
+    constructor(city:City, coord:{ lat: number, lon: number}, code:number, message:number, cnt:number, list:Weather[])  {
+        this.city = city;
+        this.coord = coord;
+        this.code = code;
+        this.message = message;
+        this.cnt = cnt;
```

Comment:
What's cnt?

Reply:
Our model is based on the JSON provided here: https://openweathermap.org/api/hourly-forecast

Reply:
Can we use different attribute names for the model layer? Making a strong correlation between the API response attribute names and the model layer could make our architecture fragile (i.e.: we'll need to big refactorings on the code every time the API response is changed).